packages/blocks/src/page-block/edgeless/components/toolbar/edgeless-toolbar.ts

```diff
@@ -543,9 +547,9 @@ export class EdgelessToolbar extends WithDisposable(LitElement) {
   }
 
   override render() {
-    if (this.edgeless.page.readonly) return nothing;
-
     const { type } = this.edgelessTool;
+    if (this.edgeless.page.readonly && type !== 'frameNavigator')
```

Comment:
Don't use this style, please stick to

`if (foo) return;`

or 

```
if (someLongCondition) {
  return;
}
```

Reply:
ok